### PR TITLE
[RAPPS-DB] ZeroBrane Studio for RAPPS

### PR DESCRIPTION
**Pull request by Fraizeraust**
ZeroBrane Studio is a Lua IDE development software that I would like to suggest it onto RAPPS Database. The base software functions work and no technical issues occurred. You can find more information such as screenshots of the program on [CORE-13726](https://jira.reactos.org/browse/CORE-13726).

### CODE REVIEW

File: ZeroBrane.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+; UTF-8
```
Comment (sanchaez) [inline ZeroBrane.txt]: This is redundant.

File: ZeroBrane.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ 
+ [Section]
+ Name = ZeroBrane Studio
+ Version = 1.70
+ Licence = MIT
```
Comment (sanchaez) [inline ZeroBrane.txt]: s/Licence/License

**Comment by Fraizeraust** [comment]
@sanchaez I edited the file, hopefully everything is alright now. By the way, can I get the SHA1 hash entry with HashTab software? If not, then how?

**Comment by sanchaez** [comment]
SHA1 is a SHA1. You can use any software that can calculate that hash.

**Comment by Fraizeraust** [comment]
Here you go, SHA1 hash added in the text file. If you find anything missing then please let me note of this and I'll fix the mistakes!

**Comment by sanchaez** [comment]
Only thing left before merging is to make SHA1 lowercase :blush:

**Comment by Fraizeraust** [comment]
Done as you requested, sir.

**Comment by sanchaez** [comment]
Perfect, thank you, @Fraizeraust!

**Pull request merged**